packages/dd-trace/test/appsec/iast/overhead-controller.spec.js

```diff
@@ -0,0 +1,112 @@
+const overheadController = require('../../../src/appsec/iast/overhead-controller')
+
+describe('Overhead controller', () => {
+  const oceContextKey = overheadController.OVERHEAD_CONTROLLER_CONTEXT_KEY
+
+  describe('Initialize OCE context', () => {
+    it('should populate oce context', () => {
+      const iastContext = {}
+      overheadController.initializeRequestContext(iastContext)
+      expect(iastContext).to.have.nested.property(overheadController.OVERHEAD_CONTROLLER_CONTEXT_KEY)
+    })
+  })
+
+  describe('Single shot operation', () => {
+    it('should not allow with no context', () => {
+      expect(overheadController.hasQuotaSingleShot(overheadController.SINGLE_SHOT_OPERATIONS[0])).to.be.false
+      expect(overheadController.hasQuotaSingleShot(overheadController.SINGLE_SHOT_OPERATIONS[0], null)).to.be.false
+      expect(overheadController.hasQuotaSingleShot(overheadController.SINGLE_SHOT_OPERATIONS[0], {})).to.be.false
+    })
+
+    describe('Report vulnerability', () => {
+      let iastContext
+
+      const SINGLE_SHOT_OPERATION = overheadController.SINGLE_SHOT_OPERATIONS.REPORT_VULNERABILITY
+
+      beforeEach(() => {
+        iastContext = {}
+        overheadController.initializeRequestContext(iastContext)
+      })
+
+      it('should populate initial context with available tokens', () => {
+        const _iastContext = {}
```

Comment:
you have already defined `let iastContext` in parent scope, why are you creating this new `_iastContext` if this test is not using the original one?